sceptre/admincentral/config/prod/peering-nextflow-dev.yaml

```diff
@@ -0,0 +1,7 @@
+template_path: VPCPeer.yaml
+stack_name: peering-nextflow-dev
+parameters:
+  PeerVPC: vpc-04c1457103adc1fdf
+  PeerVPCOwner: 035458030717
```

Comment:
i think you need to put quotes around numbers.  also can you please add comments like the ones in peering-iatlasvpc.yaml?  https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/sceptre/admincentral/config/prod/peering-iatlasvpc.yaml